File: xamarin/Xtrade/Xtrade.AndroidApp/Activities/BaseActivity.cs
namespace Xtrade.AndroidApp.Activities
{
    using Android.OS;
    using Android.Support.V7.App;
    using Android.Views;

    public class BaseActivity : AppCompatActivity
    {
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);

            if (Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop)
            {
                this.Window.AddFlags(WindowManagerFlags.DrawsSystemBarBackgrounds);
            }
        }
    }
}

File: android/Xtrade/app/src/main/java/nz/co/fendustries/xtrade/activities/BaseActivity.java
package nz.co.fendustries.xtrade.activities;

import android.os.Build;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.WindowManager;

/**
 * Created by joshuafenemore on 21/10/16.
 */
public class BaseActivity extends AppCompatActivity
{
    @Override
    protected  void onCreate(Bundle bundle)
    {
        super.onCreate(bundle);

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
        {
            this.getWindow().addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
        }
    }
}